### PR TITLE
KRACOEUS-8760:add application footer with copy right info

### PR DESCRIPTION
**Pull request by jwillia**

### CODE REVIEW

File: coeus-impl/src/main/resources/org/kuali/kra/datadictionary/krad/common/KcApplicationHeader.xml [hunk 1]
```diff
@@ -49,6 +49,22 @@
...
     </bean>
 
+    <bean id="Uif-GlobalApplicationFooter" parent="Kc-ApplicationFooter"/>
+    <bean id="Kc-ApplicationFooter" parent="Kc-ApplicationFooter-parentBean"/>
+    <bean id="Kc-ApplicationFooter-parentBean" abstract="true" parent="Uif-FooterBase" p:cssClasses="kc-applicationFooter">
+        <property name="onDocumentReadyScript" value="Kc.Global.makeApplicationFooterSticky();"/>
+        <property name="items">
+            <list>
+                <bean parent="Uif-Image" p:source="@{#ConfigProperties['krad.externalizable.images.url']}logo-kuali.png" p:altText="Kuali Logo" p:cssClasses="pull-right"/>
+                <bean parent="Uif-Message" p:messageText="Copyright 2005-2014 The Kuali Foundation. All rights reserved.[br]
```
Comment (tdurkin) [inline coeus-impl/src/main/resources/org/kuali/kra/datadictionary/krad/common/KcApplicationHeader.xml]: This should be through 2015 (not 2014)

**Comment by jwillia** [comment]
applicationfooterview

**Comment by jwillia** [comment]
updated to 2015

**Comment by tdurkin** [comment]
Sorry, Joe - the Copyright also needs to change to "Kuali, Inc."

**Comment by jwillia** [comment]
updated to "Kuali, Inc."

**Comment by leoherbie** [comment]
rebased in PR https://github.com/kuali/kc/pull/1537

**Pull request closed without merging**